### PR TITLE
DOC-4130

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/ROOT/pages/relnotes-nodejs-sdk.adoc [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+[source,bash]
+----
+$ npm install couchbase@2.3.5
+----
+
+https://docs.couchbase.com/sdk-api/couchbase-node-client-2.3.5/[API Reference]
+
+=== Fixed Issues
+
+* http://issues.couchbase.com/browse/JSCBC-342[JSCBC-3]http://issues.couchbase.com/browse/JSCBC-395[95]:
```
Comment (brett19) [inline modules/ROOT/pages/relnotes-nodejs-sdk.adoc]: The linking here is a little bit broken I think.

File: modules/ROOT/pages/relnotes-nodejs-sdk.adoc [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+
+* http://issues.couchbase.com/browse/JSCBC-356[JSCBC-356]:
+Implemented an role-based authenticator.
+* http://issues.couchbase.com/browse/JSCBC-368[JSCBC-368]: Updated
+RBAC user management with new API endpoints.
+* http://issues.couchbase.com/browse/JSCBC-372[JSCBC-372]:
+Implemented method to allow flushing of the query cache.
+* http://issues.couchbase.com/browse/JSCBC-375[JSCBC-375]: Exposed
+all libcouchbase error codes to Node.js.
+* http://issues.couchbase.com/browse/JSCBC-358[JSCBC-3]http://issues.couchbase.com/browse/JSCBC-371[71]:
```
Comment (brett19) [inline modules/ROOT/pages/relnotes-nodejs-sdk.adoc]: Here too!

File: modules/ROOT/pages/relnotes-nodejs-sdk.adoc [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+[source,bash]
+----
+$ npm install couchbase@2.3.2
+----
+
+https://docs.couchbase.com/sdk-api/couchbase-node-client-2.3.2/[API Reference]
+
+=== Fixed Issues
+
+* http://issues.couchbase.com/browse/JSCBC-352[JSCBC-3]http://issues.couchbase.com/browse/JSCBC-361[61]:
```
Comment (brett19) [inline modules/ROOT/pages/relnotes-nodejs-sdk.adoc]: Woopsies.

File: modules/ROOT/pages/relnotes-nodejs-sdk.adoc [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+Version 2.2.5 is a bug-fix release for the 2.2 serries.
+
+[source,bash]
+----
+$ npm install couchbase@2.2.5
+----
+
+=== Fixed Issues
+
+* http://issues.couchbase.com/browse/JSCBC-339[JSCBC-3]http://issues.couchbase.com/browse/JSCBC-340[40]:
```
Comment (brett19) [inline modules/ROOT/pages/relnotes-nodejs-sdk.adoc]: I'll stop now.

**Review by brett19 — CHANGES_REQUESTED** [review]
Just some minor linking failures during the conversion.

**Comment by RichardSmedley** [comment]
Changes made

**Comment by brett19** [comment]
+1

**Pull request merged**